monero-outreach-docs/translations/es/archive/triptych_es.md

```diff
@@ -0,0 +1,43 @@
+
+### _Desafíos_
+
+Ocultar al remitente en Monero no es perfecto. Diez decoys no siempre son suficientes, ya que algún TXO aleatorio puede ser identificable. Puede que con conocimiento público, los TXO, hayan sido utilizados en algún otro gasto. La existencia de las listas negras o blacklists muestra que esto puede ser un problema potencial y le da una solución a aquellos que esten motivados a esquivar y a agregar esta lista en sus monederos. Existen otras maneras de identificar los decoys en una transacción, como por ejemplo, atar los TXO de los pools y los TXO de las casas de cambio, estimando su validez en otras transacciones y utilizando tiempos de probabilidad.
+
+Imagina un alineamiento policial con un acusado injustamente y 10 decoys. Si los decoys fueran conocidos a un testigo falso, no serviría de nada proteger al acusado. Esto también se confirma con las firmas de anillo de Monero; ocultarse entre decoys solo sirve cuando los decoys son extraños y, para algunos adversarios, en particular adversarios con recursos, solo 10 decoys aleatorios pueden ser insuficiente. Y entonces, ¿por qué no usar más?
+
+Bueno, para el algoritmo MLSAG/CLSAG utilizado por Monero, tanto el tamaño de la transacción como el tiempo para procesarla crecen linealmente con la cantidad de decoys (según su complejidad algorítmica Big O). La utilización de diez decoys compensa la necesidad de privacidad con la necesidad de una cadena de bloques pequeña y un procesamiento más rápido. Incrementar el número de decoys puede ayudar a mejorar la privacidad pero también incrementa el tamano de la transacción, que de por si ya es un poco más grande que las transacciones de Bitcoin, y ralentizar la validación de los monederos.
+
+### _Triptych_
+
+Abordando este desafío encontramos a Triptych, un nuevo algoritmo para las firmas de anillo desarrollado por Sarang Noether y Brandon Goodell del Laboratorio de investigación de Monero, con la asociación externa de Arthur Blue (RandomRun). Con Triptych el tamaño de las firmas de anillo crece de manera logarítmica en vez de lineal. El crecimiento logarítmico, es muy lento, además, es el patrón de crecimiento mejor conocido sin recurrir en la necesidad de confiar en alguien para usar y luego destruir la aleatoriedad durante su configuración (tal como fue hecho para Zcash). En Triptych, los tiempos de verificación aún crecen linealmente con el número de decoys, pero la verificación puede agruparse y evaluarse utilizando algoritmos especializados los cuales ofrecen algo de aceleración.
```

Comment:
I'd use it in capitals too (because of the MRL):

_desarrollado por Sarang Noether y Brandon Goodell del **Laboratorio de Investigación de Monero**,..._

Reply:
Fixed :)